code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -113,7 +114,14 @@
 				play_sound = pick(death_sounds)
 				digestion_death(M)
```

Comment:
Honestly digestion_death() should probably handle all this but this is fine for now. Refactor later.